src/sentry/shared_integrations/client.py

```diff
@@ -70,12 +70,13 @@ def from_response(self, response, allow_text=False):
                     response.headers.get("Content-Type", ""), response.status_code
                 )
         else:
-            data = json.loads(response.text, object_pairs_hook=OrderedDict)
-
+            data = json.loads(response.text or "null", object_pairs_hook=OrderedDict)
```

Comment:
The response comes back as a 200 or 201 but the text is an empty string which `json.loads` won't accept. [This](https://stackoverflow.com/questions/48710403/load-an-empty-string-as-a-json-in-python3) suggested what I did, but lmk if there's a better way.

Reply:
I'd avoid the `json.loads()` call altogether.
```py
if not response.text:
  data = None
else:
  data = json.loads(response.text, object_pairs_hook=OrderedDict)
```

Reply:
Ok looking at it more I think you should just return early here instead of expecting `None` below. Something like
```py
if not response.text:
    return TextApiResponse("", response.headers, response.status_code)
```